apis/logging/v1/clusterlogging_types.go

```diff
@@ -199,6 +213,15 @@ type ElasticsearchSpec struct {
 	ProxySpec `json:"proxy,omitempty"`
 }
 
+// LokiStackStoreSpec is used to set up cluster-logging to use a LokiStack as logging storage.
+// It points to an existing LokiStack in the same cluster.
+type LokiStackStoreSpec struct {
+	// Name of the LokiStack resource.
+	//
+	// +required
+	Name string `json:"name"`
+}
```

Comment:
Why not simply a k8s native LocalObjectReference instead of a custom spec? I come along LORs very often lately in other operators eg the Prometheus-Operator and it comes in handy